package/system/ca-certificates/Makefile

```diff
@@ -64,7 +64,9 @@ endef
 
 define Package/ca-bundle/install
 	$(INSTALL_DIR) $(1)/etc/ssl/certs
-	cat $(PKG_INSTALL_DIR)/usr/share/ca-certificates/*/*.crt >$(1)/etc/ssl/certs/ca-certificates.crt
+	for CERTFILE in `find $(PKG_INSTALL_DIR)/usr/share/ca-certificates/ -name "*.crt" | sort`; do \
```

Comment:
Afaik this will even have the exact same result as the above using `cat` with a glob, as also the output of `sort` depends on `LC_ALL` and friends.